src/Xamarin.Android.Build.Tasks/Tasks/CompileToDalvik.cs

```diff
@@ -120,12 +127,12 @@ protected override string GenerateCommandLineCommands ()
 					var zip = Path.GetFullPath (Path.Combine (ClassesOutputDirectory, "..", "classes.zip"));
 					if (File.Exists (zip)) {
 						Log.LogDebugMessage ($"    {zip}");
-						sw.WriteLine (Path.GetFullPath (zip));
+						sw.WriteLine (Path.GetFullPath (zip).Normalize (NormalizationForm.FormC));
```

Comment:
Likewise, `ClassesOutputDirectory` is "from MSBuild" ("hardcoded" in the `.targets`, but `$(IntermediateOutputPath)` is plausibly overridable by the user and thus could contain non-ASCII characters), and as such we should *not* further normalize this value.